messages/describe/domains/errors.go

```diff
@@ -0,0 +1,5 @@
+package domains
+
+import "errors"
+
+var ErrorGetDomain = errors.New("Failed to your domain. Check your settings and try again. If the error persists, contact Azion support.")
```

Comment:
The describe word was missing? just a doubt
```suggestion
var ErrorGetDomain = errors.New("Failed to describe your domain. Check your settings and try again. If the error persists, contact Azion support.")
```

Reply:
True. It's missing a word